### PR TITLE
feat(hogql): Add matching records to actors query runner and paths query

### PR DESCRIPTION
**Pull request by webjunkie**
## Problem

We still need to port the matching records list from the old paths query.

## Changes

- add an insane amount of fields to query the matching events in the paths query
- add match record logic to actors query runner
- add record enriching to person actors

image

## How did you test this code?

- tbd

### CODE REVIEW

File: posthog/hogql_queries/insights/test/test_paths_query_runner_2.py
Comment (mariusandra) [inline posthog/hogql_queries/insights/test/test_paths_query_runner_2.py:1]: That's a lot of code 😅

**Review by mariusandra — APPROVED** [review]
This is a good amount of fields... and a bit of code 💪 😅 

I obviously couldn't verify every line reading through it now, but LGTM! 👍 

While testing, I did notice that the insight disappeared in the background if I clicked "new insight -> path -> persons -> recording", but that was also present with the old query, so 🤷

**Comment by webjunkie** [comment]
@mariusandra You don't happen to have any idea why two unrelated tests fail here? I spent quite some time now, but cannot figure it out.

**Comment by mariusandra** [comment]
Not sure regarding the unrelated ones. They seem like "standard flakes", so re-running a few times is the first thing I'd try 🤔

**Pull request merged**